cloudbuild.py.yaml

```diff
@@ -24,14 +24,22 @@ steps:
         ./run_test.sh -p
 
   - id: package_js
-    name: "node"
+    name: gcr.io/datcom-ci/node:2020-11-05
```

Comment:
here can just be "gcr.io/datcom-ci/node" which would use the latest. The versioned one is more for rollback purpose

Reply:
fixed